scripts/ci

```diff
@@ -7,6 +7,6 @@ cd $(dirname $0)
 ./validate-ci
 ./build
 ./test
+./package-webhook
```

Comment:
can we validate if the packaged webhook image that we have run in the kind cluster is the one we have built currently?

Reply:
- The image tag is [exported within environment variables](https://github.com/harvester/harvester/pull/919/commits/4e1cb34ddf118208d73e7fcf9dad8c8d90f41817#diff-2123bcafa82cdd48a07cc765c7a8207cdca8b1ab9df3e1f61a8fc3802479c450R9-R11).
- The image will [be pre-loaded into the kind cluster](https://github.com/harvester/harvester/pull/919/commits/4e1cb34ddf118208d73e7fcf9dad8c8d90f41817#diff-90be7bf703b981c58342ad8cd558dc55d3fe35411b3c7271593ee8000e47e895R127-R140).
- The helm chart [is patched with the CI image](https://github.com/harvester/harvester/pull/919/commits/4e1cb34ddf118208d73e7fcf9dad8c8d90f41817#diff-fd9115776c3c5c579b9bf3b1d7d1ad7855eb7cc64d3f4cf832de39cb70b6c666R43-R52).
- The integration test code checks if `harvester-webhook` pods are ready before testing.

Reply:
in that case, do u think would be better for us to change the image pull policy to `never`, so it should either use the imported image or fail if the image does not exist.

Reply:
Thanks, [Update the policy to `Never`](https://github.com/harvester/harvester/pull/919/commits/a0ca5030d477ccb0fdb57729f6f3b03c0f41e801).